### PR TITLE
[podreloader] distroless

### PR DESCRIPTION
**Pull request by vitaliy-sn**
## Description

The podreloader docker image is now distroless.

## Tests

<details>

```shell
v.snurnitsin@thinkpad:/tmp/tmp$ docker save dev-registry.deckhouse.io/sys/deckhouse-oss@sha256:bfe3f35dd56fcc08334fffcb54294af96549fe53d4279b2eba0e03b3d1cc32c1 > 1.tar
v.snurnitsin@thinkpad:/tmp/tmp$ tar xvf 1.tar 
30526b51275d9a1659f5348b7aa34b0c228d15edfd91ab97560cd73cfeced9ac/
30526b51275d9a1659f5348b7aa34b0c228d15edfd91ab97560cd73cfeced9ac/VERSION
30526b51275d9a1659f5348b7aa34b0c228d15edfd91ab97560cd73cfeced9ac/json
30526b51275d9a1659f5348b7aa34b0c228d15edfd91ab97560cd73cfeced9ac/layer.tar
5b6d519671356c1db31c2b33fd2ddda7d0e6233cf4a4c8aa498d804e8778f396/
5b6d519671356c1db31c2b33fd2ddda7d0e6233cf4a4c8aa498d804e8778f396/VERSION
5b6d519671356c1db31c2b33fd2ddda7d0e6233cf4a4c8aa498d804e8778f396/json
5b6d519671356c1db31c2b33fd2ddda7d0e6233cf4a4c8aa498d804e8778f396/layer.tar
6f3b47e4743716867338f986e9d610d14717d6e1214f013b97634a512a368d1a.json
baef073e354c4355169c716d86bc41490550d7329ffbeb3f61282f662c6cfa4f/
baef073e354c4355169c716d86bc41490550d7329ffbeb3f61282f662c6cfa4f/VERSION
baef073e354c4355169c716d86bc41490550d7329ffbeb3f61282f662c6cfa4f/json
baef073e354c4355169c716d86bc41490550d7329ffbeb3f61282f662c6cfa4f/layer.tar
manifest.json
v.snurnitsin@thinkpad:/tmp/tmp$ tree -h
[4.0K]  .
├── [ 41M]  1.tar
├── [4.0K]  30526b51275d9a1659f5348b7aa34b0c228d15edfd91ab97560cd73cfeced9ac
│   ├── [ 482]  json
│   ├── [2.5K]  layer.tar
│   └── [   3]  VERSION
├── [4.0K]  5b6d519671356c1db31c2b33fd2ddda7d0e6233cf4a4c8aa498d804e8778f396
│   ├── [2.6K]  json
│   ├── [ 38M]  layer.tar
│   └── [   3]  VERSION
├── [5.5K]  6f3b47e4743716867338f986e9d610d14717d6e1214f013b97634a512a368d1a.json
├── [4.0K]  baef073e354c4355169c716d86bc41490550d7329ffbeb3f61282f662c6cfa4f
│   ├── [ 406]  json
│   ├── [3.3M]  layer.tar
│   └── [   3]  VERSION
└── [ 343]  manifest.json

4 directories, 12 files
v.snurnitsin@thinkpad:/tmp/tmp$ cd 5b6d519671356c1db31c2b33fd2ddda7d0e6233cf4a4c8aa498d804e8778f396
v.snurnitsin@thinkpad:/tmp/tmp/5b6d519671356c1db31c2b33fd2ddda7d0e6233cf4a4c8aa498d804e8778f396$ tar xvf layer.tar 
reloader
v.snurnitsin@thinkpad:/tmp/tmp/5b6d519671356c1db31c2b33fd2ddda7d0e6233cf4a4c8aa498d804e8778f396$ ldd reloader 
        not a dynamic executable
v.snurnitsin@thinkpad:/tmp/tmp/5b6d519671356c1db31c2b33fd2ddda7d0e6233cf4a4c8aa498d804e8778f396$
```

</details>

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: podreloader
type: feature
summary: The podreloader docker image is now distroless
impact: pod-reloader pod will be restarted
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've started the workflow for label 'e2e/run/aws'...

File: modules/465-pod-reloader/images/pod-reloader/werf.inc.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+artifact: {{ .ModuleName }}/reloader-artifact
+from: {{ .Images.BASE_GOLANG_19_ALPINE }}
+shell:
+  install:
+    - apk add --no-cache git
+    - mkdir -p /src
+    - cd /src
+    - git clone --depth 1 -b v1.0.42 https://github.com/stakater/Reloader.git .
```
Comment (RomanenkoDenys) [inline modules/465-pod-reloader/images/pod-reloader/werf.inc.yaml:9]: SOURCE_REPO please

File: modules/465-pod-reloader/images/pod-reloader/werf.inc.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    - GOOS=linux GOARCH=amd64 CGO_ENABLED=0 go build -ldflags="-s -w" -o /reloader main.go
```
Comment (RomanenkoDenys) [inline modules/465-pod-reloader/images/pod-reloader/werf.inc.yaml:10]: GOPROXY needed

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
some nits

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @vitaliy-sn. I've started the workflow for label 'e2e/run/aws'...

File: modules/465-pod-reloader/images/pod-reloader/werf.inc.yaml (hunk 1 above)
Comment (vitaliy-sn) [reply modules/465-pod-reloader/images/pod-reloader/werf.inc.yaml]: done

**Review by name212 — APPROVED** [review]

**Review by RomanenkoDenys — APPROVED** [review]
LGTM

**Pull request merged**